spec.html

```diff
@@ -8928,12 +8951,10 @@ <h1>ArrayCreate ( _length_ [ , _proto_ ] )</h1>
           1. Assert: _length_ is an integer Number &ge; 0.
           1. If _length_ is *-0*, set _length_ to *+0*.
           1. If _length_ &gt; 2<sup>32</sup> - 1, throw a *RangeError* exception.
-          1. If _proto_ is not present, set _proto_ to %Array.prototype%.
-          1. Let _A_ be a newly created Array exotic object.
-          1. Set _A_'s essential internal methods to the default ordinary object definitions specified in <emu-xref href="#sec-ordinary-object-internal-methods-and-internal-slots"></emu-xref>.
-          1. Set _A_.[[DefineOwnProperty]] as specified in <emu-xref href="#sec-array-exotic-objects-defineownproperty-p-desc"></emu-xref>.
+          1. If _proto_ is not present, set _proto_ to the intrinsic object %Array.prototype%.
```

Comment:
is "the intrinsic object" needed here? the percent syntax seems to cover that.

Reply:
Will remove.